app/app/settings.py

```diff
@@ -176,7 +176,9 @@
         'anon': '1000/day',
     },
     'DEFAULT_PERMISSION_CLASSES': ['rest_framework.permissions.DjangoModelPermissionsOrAnonReadOnly'],
-    'DEFAULT_AUTHENTICATION_CLASSES': []
+    'DEFAULT_AUTHENTICATION_CLASSES': [
```

Comment:
where did this come from?  its not from https://github.com/gitcoinco/web/pull/1839/files